### ISSUE TITLE
Fails to execute in Gradle (NoSuchMethodError in commons-io) (reopen #2509)

### ISSUE THREAD

**Issue by shevek** [dependency-check/DependencyCheck#3468]
**Describe the bug**
```
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: org.owasp.dependencycheck.exception.ExceptionCollection: One or more exceptions occurred during analysis:
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   InitializationException: Unexpected Exception
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]           caused by NoSuchMethodError: org.apache.commons.io.IOUtils.length([Ljava/lang/Object;)I
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   InitializationException: Unexpected Exception
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]           caused by NoSuchMethodError: org.apache.commons.io.IOUtils.length([Ljava/lang/Object;)I
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   InitializationException: Unexpected Exception
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]           caused by NoSuchMethodError: org.apache.commons.io.IOUtils.length([Ljava/lang/Object;)I
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.owasp.dependencycheck.Engine.analyzeDependencies(Engine.java:662)
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.owasp.dependencycheck.Engine$analyzeDependencies$0.call(Unknown Source)
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   at org.owasp.dependencycheck.gradle.tasks.AbstractAnalyze.analyze(AbstractAnalyze.groovy:90)
2021-06-23T11:47:02.002-0700 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   ... 120 more
```

This reopens #2509 because of 39f6a0b770a78ba3a0fbca9fa87e25968e429515 which re-replaced the Guava calls with IOUtils calls. This was previously fied in #2587 but the commit referenced above is a regression.

**Version of dependency-check used**
The problem occurs using version 6.2.2 of the gradle plugin with Gradle 6.8.3

**Expected behavior**
Don't crash. :grin: 

**Additional context**
See PR #2587 for the fix.

**Comment by shevek** [comment]
Alternatively reimplement the method from ioutils within this library, if one wants to avoid calling guava?

**Comment by chadlwilson** [comment]
@jeremylong looks very old, fixed in #2509  The dependency clashes with other plugins are an ongoing 'joy' though :)
